src/Hie/Cabal/Parser.hs

```diff
@@ -121,9 +121,10 @@ pathMain i p m o a =
     <|> (skipBlockLine i >> pathMain i p m o a)
     <|> pure
       ( map (<//> m) p
-          <> [ p' <//> (o' <> ".hs")
+          <> [ p' <//> (o'' <> ".hs")
                | p' <- p,
-                 o' <- filter (`notElem` a) o
+                 o' <- filter (`notElem` a) o,
+                 let o'' = T.replace "." "/" o'
```

Comment:
Is that platform independent? So, works on windows?

Reply:
Yeah, i hope so or it will not work for me 😄 
well, `hie.yaml` uses unix paths in the config and it works, right?, but i will check just in case

Reply:
You are right, those paths get normalised before usage, iirc.